src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,677 @@ description: Learn how to use images in Astro.
+
+As an alternative, you can wrap these components in another Astro component for reuse. For example, you could create a component for your blog post images:
+
+```astro title="src/components/BlogPostImage.astro"
+---
+import { Image } from 'astro:assets';
+
+const {src, ...attrs} = Astro.props;
+---
+<Image src={src} {...attrs} />
+
+<style>
+  img :global(img), svg {
+    margin-block: 2.5rem;
+    border-radius: 0.75rem;
+  }
+</style>
+```
+
+### `<img>`
+
+The [Astro template syntax](/en/core-concepts/astro-syntax/) also supports writing an `<img>` tag directly, with full control over its final output. Local images will be optimized.
+
+It accepts all HTML `<img>` tag properties, and the only required property is `src`.
+
+#### Local images in `src/`
+
+Local images must be **imported from the relative path** from the existing `.astro` file, or configure and use an [import alias](/en/guides/aliases/). Then, you can access the image's `src` and other properties to use in the `<img>` tag.
+
+```astro "myDog.src"
+---
+// import local images
+import myDog from `../../images/pets/local-dog.jpg`
+---
+// access the image properties
+<img src={myDog.src} alt="A barking dog." />
+```
+
+Imported image assets match the following signature:
+
+```ts
+interface ImageMetadata {
+  src: string;
+  width: number;
+  height: number;
+  format: string;
+}
+```
+
+#### Images in `public/`
+For images located within `public/` use the image's **file path relative to the public folder** as the `src` value:
 
-<!-- Local image stored at src/images/rocket.svg -->
-<img src={rocket} alt="A rocketship in space."/>
+```astro '"/images/public-cat.jpg"'
+<img src="/images/public-cat.jpg" alt="A sleeping cat.">
 ```
 
-### In Markdown files
+#### Remote images
 
-You can use standard Markdown `![]()` syntax or standard HTML `<img>` tags in your `.md` files for images located in your `public/` folder, or remote images on another server.
+For remote images, use the image's **full URL** as the `src` value:
 
-If you cannot keep your images in `public/`, we recommend enabling [experimental support for assets](/en/guides/assets/), or using the `.mdx` file format, which allows you to combine imported components with Markdown-like syntax. Use the [MDX integration](/en/guides/integrations-guide/mdx/) to add support for MDX to Astro.
+```astro '"https://example.com/remote-cat.jpg"'
+<img src="https://example.com/remote-cat.jpg" alt="A sleeping cat.">
+```
+
+### Choosing `<Image />` vs `<img>`
+
+The `<Image />` component optimizes your image and infers width and height (of local images) based on the original aspect ratio to avoid CLS. But, it only works with certain formats and does not provide a `<picture>` element, nor does it support `srcset`.
+
+Use the HTML `<img>` element when you cannot use the `<Image />` component, for example:
+  - for unsupported image formats, like `.svg`
```

Comment:
You can pass `.svg` to `<Image>`, they won't be processed but you still get the benefits regarding CLS, loading/decoding and alt.